test/crypto.test.ts

```diff
@@ -0,0 +1,86 @@
+import 'mocha'
+import * as assert from 'assert'
+import * as helpers from '../src/crypto'
+
+if (typeof describe === 'function') {
+  describe('crypto helpers (node)', function () {
+    runCryptoTests({ describe, it })
+  })
+}
+
+export function runCryptoTests (args: {describe: any, it: any}) {
+  const { describe, it } = args
+
+  describe('generateToken', function () {
+    it('generates a random 18-byte token', function () {
+      assert.equal(helpers.generateToken().length, 18)
+    })
+  })
+
+  describe('generateSharedSecretFromToken', function () {
+    it('generates the expected secret', async function () {
+      const seed = Buffer.alloc(32)
+      const gotSecret = await helpers.generateSharedSecretFromToken(seed, Buffer.from('foo bar'))
+      const wantSecret = Buffer.from('ImTMJmMhjK4VSEyhjCUwlUnatWGB+Pm/UMOrbE6ieWE=', 'base64')
+      assert.deepEqual(gotSecret, wantSecret)
+    })
+  })
+
+  describe('generateRandomCondition', function () {
+    it('generates a random 32-bit condition', function () {
+      assert.equal(helpers.generateRandomCondition().length, 32)
+    })
+  })
+
+  describe('generatePskEncryptionKey', function () {
+    it('generates the expected key', async function () {
+      const secret = Buffer.from('foo')
+      const gotKey = await helpers.generatePskEncryptionKey(secret)
+      const wantKey = Buffer.from('zfOGMU/uY+EOUCE6tN3WhE77bF/N6pS0wSOmMgEAMEA=', 'base64')
+      assert.deepEqual(gotKey, wantKey)
+    })
+  })
+
+  describe('generateFulfillmentKey', function () {
+    it('generates the expected key', async function () {
+      const secret = Buffer.from('foo')
+      const gotKey = await helpers.generateFulfillmentKey(secret)
```

Comment:
I think your editor is mistaken (possibly since its typescript) -- those `assert` functions are only deprecated in legacy (non-`strict`) mode (see https://nodejs.org/docs/latest-v12.x/api/assert.html#assert_strict_mode). Typescript emits `"use strict"`.

Reply:
I see, thanks.